src/Codeception/Command/Run.php

```diff
@@ -443,6 +436,9 @@ protected function matchSingleTest($suite, $config)
                 }
                 $testsPath = $config['paths']['tests'] . DIRECTORY_SEPARATOR . $suiteConfig['path'];
                 if ($suiteConfig['path'] === '.') {
+                    if ($config['paths']['tests'] === '.') {
+                        return ['', $s, $suite];
```

Comment:
I don't like this line at all. 

This code is inside `matchSingleTest` method and the third element in result array is supposed to be a path to specific test file.
This condition doesn't even check if `$suite` is a path.

Reply:
The type of check you describe was not there in the past, nor in this function (line 444), nor in `matchTestFromFilename`. They both just do a `preg_match` on the input and return the result.

`$suite` is never validated as a path in any other code branch.

Reply:
anyway, I've removed this code and put back the original hack at the start of the function